psutil/_psaix.py

```diff
@@ -120,12 +126,16 @@ def get_procfs_path():
 
 
 def virtual_memory():
+    if not hasattr(cext, 'virtual_mem'):
+        _not_supported
```

Comment:
this should be `_not_supported()` (and same below)

Reply:
Oops! Thanks for catching